src/main/scala/io/scala/modules/layout/Header.scala

```diff
@@ -27,29 +27,35 @@ object Header {
   def render =
     headerTag(
       logo,
-      inlineLinks,
-      burgerMenu
+      div(
+        display.flex,
+        alignItems.center,
+        inlineLinks,
+        burgerMenu,
+        Links.button("Tickets", "https://www.billetweb.fr/scalaio-paris-2024-epita")(padding := "0.5rem 0.5rem")
```

Comment:
How about redirecting to https://www.scala.io/#tickets instead in order to avoid having users confused by two different shopping screens?

Reply:
Yep that's also a good solution

Reply:
9fb5a484e32c7cab15a0e4fdde39613bca7d063e